api/CcsSso.Core.Service/External/UserProfileService.cs

```diff
@@ -227,6 +227,10 @@ public async Task<UserEditResponseInfo> CreateUserAsync(UserProfileEditRequestIn
       _dataContext.Party.Add(party);
 
       await _dataContext.SaveChangesAsync();
+      if(isNewOrgAdmin==true)
+      {
+          await AddUserToDefaultGroup(organisation.Id, party.User.Id);
```

Comment:
AddUserToDefaultAdminGroup

Reply:
Updated the method name to AddUserToDefaultAdminGroup